packages/pfsense/data_stream/log/elasticsearch/ingest_pipeline/dhcp.yml

```
@@ -13,47 +19,36 @@ processors:
         ON: 'on %{IP:client.address} to %{MAC_ADDRESS:client.mac} \(%{HOSTNAME:pfsense.dhcp.hostname}\)'
         FOR: 'for %{IP:client.address} \(%{IP:server.address}\)? from %{MAC_ADDRESS:client.mac} \(%{HOSTNAME:pfsense.dhcp.hostname}\)'
         MIDDLE: '(?:%{FROM}|%{ON}|%{FOR})'
-  - set:
-      field: event.kind
-      value: event
-  - append:
-      field: event.category
-      value: network
-      allow_duplicates: false
-  - append:
-      field: event.type
-      value: connection
-      allow_duplicates: false
-  - append:
-      field: event.type
-      value: protocol
-      allow_duplicates: false
+        DUID: '[0-9a-f]{2}(:[0-9a-f]{2})+'
```

DUIDs are specified on top of UID which allows uppercase A-F.
```suggestion
        DUID: '(?i)[0-9a-f]{2}(:[0-9a-f]{2})+'
```

---

PFsense appear to be all lowercase DUIDs but i'll add it to be safe